### PR TITLE
fix: Fixed some design issues 

### PR DESCRIPTION
**Pull request by bandanasahu123**
Fixed some design issues  and added the navigation in tag related discussion

### CODE REVIEW

File: projects/discussion-ui/src/lib/components/discuss-all/discuss-all.component.scss [hunk 1]
```diff
@@ -1,135 +1,24 @@
-// @import 'ws-common';
-// @import 'ws-vars';
-// @import 'ws-mixins';
-// @import "node_modules/bootstrap/scss/bootstrap";
```
Comment (christyfernandes) [inline projects/discussion-ui/src/lib/components/discuss-all/discuss-all.component.scss:4]: Why these CSS are removed ?

File: projects/discussion-ui/src/lib/components/tag-all-discussion/tag-all-discussion.component.html [hunk 2]
```diff
@@ -9,10 +9,11 @@ <h2 class="margin-remove-bottom tagtitle" [ngStyle]="getBgColor(tagName)">{{tagN
         <div class="flex flex-1 flex-column margin-fix">
             <lib-app-loader *ngIf="fetchSingleCategoryLoader"></lib-app-loader>
 
-            <ng-container *ngIf="similarPosts && similarPosts.length > 0">
-                <ng-container *ngFor="let data of similarPosts">
+            <ng-container *ngIf="similarPosts  && similarPosts.length > 0">
+                <div (click)="navigateToDiscussionDetails(data);logTelemetry($event)"
```
Comment (christyfernandes) [inline projects/discussion-ui/src/lib/components/tag-all-discussion/tag-all-discussion.component.html:13]: This was not there earlier?

**Review by christyfernandes — CHANGES_REQUESTED** [review]

File: projects/discussion-ui/src/lib/components/discuss-all/discuss-all.component.scss (hunk 1 above)
Comment (bandanasahu123) [reply projects/discussion-ui/src/lib/components/discuss-all/discuss-all.component.scss:4]: these CSS are unused. I checked what class are there inside html that i kept. other unused css i removed.

File: projects/discussion-ui/src/lib/components/tag-all-discussion/tag-all-discussion.component.html (hunk 2 above)
Comment (bandanasahu123) [reply projects/discussion-ui/src/lib/components/tag-all-discussion/tag-all-discussion.component.html:13]: No navigation was there in tag related discussion. I added now.

**Pull request merged**